src/components/CarContainer.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { CarRowInfo } from "../interfaces/Car";
+import { IMG_URL } from "../utils/index";
+import { FlexColumn, H3, ValidationText } from "./styled";
+import { CarInfo } from "./CarInfo.component";
+import { ApolloError } from "@apollo/client";
```

Comment:
please add a config to your linter to have external libraries imported at the top of the file, then custom components/files

Reply:
I've just fixed this in the latest commit! Thanks